app/views/shared/registrations/_action_links_panel.html.erb

```diff
@@ -4,7 +4,7 @@
   </h2>
   <ul>
     <!-- TODO: This `can?` will need to be updated when we build NewRegistration -->
```

Comment:
Can we delete this TODO now?

Reply:
Not really 🤔 We still don't have a `NewRegistration` 🤷‍♀

Reply:
Ok, can we update it then?

```suggestion
    <!-- TODO: This `display_resume_link_for?` will need to be updated when we build NewRegistration -->
```